packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -420,19 +420,18 @@ export const tide = (axios, site, config) => ({
     return pageData
   },
 
-  getPreviewPage: async function (contentType, uuid, revisionId, section, params, headersConfig) {
-    if (revisionId === 'latest') {
-      params.resourceVersion = 'rel:working-copy'
-    } else {
-      params.resourceVersion = `id:${revisionId}`
-    }
+  getPageByPreviewLink: async function (path, section) {
+    const params = {}
+    const { 2: contentType, 3: uuid, 4: revisionId } = path.split('/')
+
+    params.resourceVersion = (revisionId === 'latest') ? 'rel:working-copy' : `id:${revisionId}`
 
     const pathData = {
       entity_type: 'node',
       bundle: contentType,
       uuid: uuid
     }
-    const entity = await this.getEntityByPathData(pathData, params, headersConfig)
+    const entity = await this.getEntityByPathData(pathData, params)
```

Comment:
Can we add `headersConfig ` back? It will be used for passing extra headers like request id.

Reply:
Added